internal/db/schema/migrations/oss/postgres/99/03_worker_authentication.up.sql

```diff
@@ -67,9 +87,15 @@ create table worker_auth_authorized(
       references kms_data_key_version (private_id)
       on delete restrict
       on update cascade,
-  nonce bytea
+  nonce bytea,
+  version wt_version
```

Comment:
I'm just thinking.... do we want to make this data immutable which would remove the need for a version.  Then we just delete/write new entries for the ID when the creds rotate?

Reply:
I like that idea!